roboflow/core/workspace.py

```diff
@@ -277,8 +279,13 @@ def active_learning(
             else conditionals["maximum_size_requirement"]
         )
 
+        # check if inference_model references endpoint or local
+        local = "http://localhost:9001/" if use_localhost else None
```

Comment:
food for thought, we could wrap up the `"http://localhost:9001/"` in the `version` object if we don't want to give users the option to specify alternative IP addresses or ports.

I know Dwayne will sometimes call a direct IP instead of the localhost but not sure what would be best.

Reply:
Ok for now